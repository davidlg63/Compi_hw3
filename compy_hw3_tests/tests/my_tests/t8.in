bool foo( byte v,bool d){
    bool a = not true;
    auto dd = not (false);
    return not a;
}

bool t(){
    return not (1);
}