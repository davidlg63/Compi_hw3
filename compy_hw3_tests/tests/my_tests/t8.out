---end scope---
v BYTE -1
d BOOL -2
a BOOL 0
dd BOOL 1
line 8: type mismatch
